src/components/NavBar.vue

```diff
@@ -46,37 +29,45 @@
       />
       <v-spacer />
 
-      <nav-button
+      <amb-button
+        with-badge
+        data-cy="timeline-icon"
+        is-icon
+        icon="history"
+        btn-color="secondary"
+        to="timeline"
+      />
+
+      <amb-button
+        with-badge
+        data-cy="download-off"
+        is-icon
+        icon="download-off"
+        btn-color="secondary"
+        v-if="!isEdgeConnected"
+        @click="$router.push('edge-connect')"
+      />
+
+      <!-- test -->
+      <amb-button
```

Comment:
@vickywane why are you adding back the heart icon? It was removed in a previous commit, because it does not do anything useful.

Reply:
I think i started working on this feature when the icons were still in use. I would remove them

Reply:
Which is why you need to `pull` from upstream main frequently and be careful with merging so you don't override more recent work.